core/src/test/java/com/xmartlabs/base/core/DateHelperTest.java

```diff
@@ -46,6 +56,139 @@ public void getListOfDaysBetweenTwoWeeks() {
 
     List<LocalDate> listOfDaysBetweenTwoDates = DateHelper.getListOfDaysBetweenTwoDates(firstDayOfWeek, end);
 
-    Assert.assertEquals(expectedList, listOfDaysBetweenTwoDates);
+    assertThat(expectedList, equalTo(listOfDaysBetweenTwoDates));
+  }
+
+  @Test
+  public void getFirstDayOfWeekMonday() {
+    LocalDate expectedMonday = LocalDate.of(2017, Month.APRIL, 10);
+    LocalDate monday = LocalDate.of(2017, Month.APRIL, 10);
+    LocalDate actualMonday = DateHelper.getFirstDayOfWeek(monday);
+
+    assertThat(actualMonday, equalTo(expectedMonday));
+  }
+
+  @Test
+  public void getFirstDayOfWeekThursday() {
+    LocalDate expectedMonday = LocalDate.of(2017, Month.APRIL, 10);
+    LocalDate thursday = LocalDate.of(2017, Month.APRIL, 13);
+    LocalDate actualMonday = DateHelper.getFirstDayOfWeek(thursday);
+
+    assertThat(actualMonday, equalTo(expectedMonday));
+  }
+
+  @Test
+  public void getFirstDayOfWeekPreviousWeek() {
+    LocalDate notExpectedMonday = LocalDate.of(2017, Month.APRIL, 10);
+    LocalDate thursday = LocalDate.of(2017, Month.APRIL, 9);
+    LocalDate actualMonday = DateHelper.getFirstDayOfWeek(thursday);
+
+    assertThat(actualMonday, not(equalTo(notExpectedMonday)));
+  }
+
+  @Test
+  public void getFirstDayOfWeekNextWeek() {
+    LocalDate notExpectedMonday = LocalDate.of(2017, Month.APRIL, 10);
+    LocalDate tuesdayNext = LocalDate.of(2017, Month.APRIL, 18);
```

Comment:
What do you think of using constants for those values? For example year and month are repeated in almost all method of this class.